service_configuration_lib/spark_config.py

```diff
@@ -131,8 +131,9 @@ def get_aws_credentials(
     profile_name: Optional[str] = None,
     session: Optional[boto3.Session] = None,
     aws_credentials_json: Optional[str] = None,
-    assume_web_identity: bool = False,
-    session_duration: int = 43200,
+    assume_aws_role_arn: Optional[str] = None,
+    session_duration: int = 3600,
+    assume_role_user_creds_file: str = '/nail/etc/spark_role_assumer/spark_role_assumer.yaml',
```

Comment:
hmm, it's a little unfortunate that we're leaking a Yelp-only path here - but apart from providing no default or doing a bit of srv-configs shenanigans, I guess there's not much we could do otherwise